crates/bevy_asset/src/lib.rs

```diff
@@ -1025,26 +1014,24 @@ mod tests {
             assert_eq!(sub_text.text, "hello");
             let (sub_text_load, sub_text_deps, sub_text_rec_deps) =
                 asset_server.get_load_states(sub_text_id).unwrap();
-            assert_eq!(sub_text_load, LoadState::Loaded);
-            assert_eq!(sub_text_deps, DependencyLoadState::Loaded);
-            assert_eq!(sub_text_rec_deps, RecursiveDependencyLoadState::Loaded);
+            assert!(sub_text_load.is_loaded());
+            assert!(sub_text_deps.is_loaded());
+            assert!(sub_text_rec_deps.is_loaded());
 
             let d_id = c_text.dependencies[0].id();
             let d_text = get::<CoolText>(world, d_id);
             let (d_load, d_deps, d_rec_deps) = asset_server.get_load_states(d_id).unwrap();
             assert!(d_text.is_none(), "d component should not exist yet");
-            assert_eq!(d_load, LoadState::Loading);
-            assert_eq!(d_deps, DependencyLoadState::Loading);
-            assert_eq!(d_rec_deps, RecursiveDependencyLoadState::Loading);
+            assert!(d_load.is_loading());
+            assert!(d_deps.is_loading());
+            assert!(d_rec_deps.is_loaded());
```

Comment:
This line seems to have been changed accidentally as well.

Reply:
Thanks! By the way, you can review it with the status "Request Changes" instead of approving.